src/components/NumberInput/NumberInput.js

```diff
@@ -293,13 +293,19 @@ export default class NumberInput extends Component {
                   <button
                     className={`${prefix}--number__control-btn up-icon`}
                     {...buttonProps}
-                    onClick={evt => this.handleArrowClick(evt, 'up')}>
+                    onClick={evt => this.handleArrowClick(evt, 'up')}
+                    aria-label="increment number"
```

Comment:
Oh should this be translatable (e.g. via prop) BTW...?

Reply:
yes good call @asudoh, added that feature